homepage.css

```diff
@@ -31,7 +31,7 @@ ul{
 }
 
 li{
-	float:right;
+	float:left;
```

Comment:
15/08/2021: Make the link in the nav bar to be in the right side. Because I released the nav bar in the right look a bit empty so I will make the logo to be in the left and the link to be in the right side so the navigation bar to look more balance.